src/data/eth-upgrade-articles.ts

```diff
@@ -26,24 +33,25 @@ export const dannyArticles = [
   },
 ]
 
-export const benArticles = [
+/** Articles by Ben Edgington (PegaSys, ConsenSys) */
+export const benArticles: Article[] = [
   {
-    title: "What’s New in Eth2 – #88",
+    title: "What’s New in Eth2 – #92",
     description: "6 May 2022",
     link: "https://hackmd.io/@benjaminion/eth2_news/https%3A%2F%2Fhackmd.io%2F%40benjaminion%2Fwnie2_220506",
   },
   {
-    title: "What’s New in Eth2 – #88",
+    title: "What’s New in Eth2 – #91",
     description: "8 April 2022",
     link: "https://hackmd.io/@benjaminion/eth2_news/https%3A%2F%2Fhackmd.io%2F%40benjaminion%2Fwnie2_220408",
   },
   {
-    title: "What’s New in Eth2 – #88",
+    title: "What’s New in Eth2 – #90",
     description: "25 March 2022",
     link: "https://hackmd.io/@benjaminion/eth2_news/https%3A%2F%2Fhackmd.io%2F%40benjaminion%2Fwnie2_220325",
   },
   {
-    title: "What’s New in Eth2 – #88",
+    title: "What’s New in Eth2 – #89",
     description: "11 March 2022",
     link: "https://hackmd.io/@benjaminion/eth2_news/https%3A%2F%2Fhackmd.io%2F%40benjaminion%2Fwnie2_220311",
   },
```

Comment:
Also, pls, rename this file to `ethUpgradeArticles` as we are trying to have consistent file names.

Reply:
👍